src/python/turicreate/test/test_style_transfer.py

```diff
@@ -311,22 +311,23 @@ def test_state(self):
 class StyleTransferGPUTest(unittest.TestCase):
     @classmethod
     def setUpClass(self):
-        self.style_feature = self.content_feature = 'image'
-        self.style_sf = _get_data(feature=self.style_feature)
-        self.content_sf = _get_data(feature=self.content_feature)
+        self.style_sf = _get_data('image')
+        self.content_sf = _get_data('image')
 
 
     def test_gpu_save_load_export(self):
         old_num_gpus = tc.config.get_num_gpus()
         gpu_options = set([old_num_gpus, 0, 1])
         for in_gpus in gpu_options:
             tc.config.set_num_gpus(in_gpus)
-            model = tc.style_transfer.create(self.style_sf, self.content_sf, max_iterations=1)
+            original_model = tc.style_transfer.create(self.style_sf, self.content_sf, max_iterations=1)
             for out_gpus in gpu_options:
                 with test_util.TempDirectory() as path:
-                    model.save(path)
+                    original_model.save(path)
                     tc.config.set_num_gpus(out_gpus)
                     model = tc.load_model(path)
+
+                with test_util.TempDirectory() as path:
```

Comment:
What is the reason to delete the model path out from under the model?  I don't think this is supported.

Reply:
Good catch. Ironically, this is basically the same bug I was fixing.

I've updated the code to export while the model is still in scope.